sdk/identity/identity-web/src/index.ts

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { MsalAuthorizationCode } from "../../identity/src/msal/nodeFlows/msalAuthorizationCode";
+import { MsalFlow } from "../../identity/src/msal/flows";
```

Comment:
I was wondering let's avoid taking dependencies on source files. Let's import these from what we have exported publicly from identity package. Else we might run into similar issues as the plugin packages. @sadasant

Reply:
To avoid this is to either copy the code or to make the separate “common” repo that you were working on. To make this PR simpler, I will keep Will’s approach for now.